debug.md

```diff
@@ -0,0 +1,39 @@
+# assisted-service
+
+## Debug Setup
+
+To debug the application first set the `DEBUG` environment variable to any nonempty value
+
+```shell
+export DEBUG=true
+```
+
+The default remote debug port is 40000 but its configurable by setting the `DEBUG_PORT` env variable:
+
+```shell
+export DEBUG_PORT=8765
+```
+
+Compile the code with debug information and patch the image:
+
+```shell
+skipper make update-debug-minimal
+```
+
+Compile the code with debug information, patch the image and push to your local k8s:
```

Comment:
Maybe add a note about how the original image has to be built first, and then it gets patched with just debug binaries?

Reply:
Actually you don't have to build the original image you can just download it
The only problematic case is when you changed the image by setting the SERVICE environment variable and you never pushed it or the image in the remote registry is too old.
I think that when the image is not available we can easily just build it, but when the image itself change i'm not sure how to verify that